components/ui/Tag.jsx

```diff
@@ -0,0 +1,12 @@
+function Tag(props) {
+  return (
+    <a
+      className={`h-fit rounded-4.75 bg-light-grey py-0.75 px-2.5 font-Neufile text-lg font-normal tracking-spacing-lg text-grey ${props.style}`}
+      href={props.link}
+    >
+      {props.children}
+    </a>
+  );
+}
+
+export default Tag;
```

Comment:
Good Job! But to be consistent I suggest you put export on the function  declaration . for instance 
do export default function Tag(){}

Reply:
Change applied